eZ/Publish/Core/Repository/TrashService.php

```diff
@@ -369,6 +377,7 @@ protected function buildDomainTrashItemObject(Trashed $spiTrashItem, Content $co
                 'sortField' => $spiTrashItem->sortField,
                 'sortOrder' => $spiTrashItem->sortOrder,
                 'trashed' => isset($spiTrashItem->trashed) ? new DateTime('@' . $spiTrashItem->trashed) : new DateTime('@0'),
+                'parentLocation' => $this->proxyDomainMapper->createLocationProxy($spiTrashItem->parentId),
```

Comment:
I kinda expected here `LocationService::load` which would create lazy instance (w/o performing actual load), but maybe I'm missing the point or not possible, or both.